packages/components/src/Actions/ActionDropdown/ActionDropdown.component.md

```diff
@@ -0,0 +1,166 @@
+# ACTION DROPDOWN COMPONENT
+
+This component display a dropdown with items in it. It used react-bootstrap component DropdownButton, MenuItem.
+
+## RENDERING
+
+#### CORE
+The main component used DopdownButton from react-bootstrap
+```javascript
+	<DropdownButton
+			title={title}
+			bsStyle={style}
+			role="button"
+			onSelect={onItemSelect}
+			className={classNames(theme['tc-dropdown-button'], 'tc-dropdown-button')}
+			{...rest}
+		>
+			{chooseMenuItemRendering(getComponent, items, components)}
+		</DropdownButton>
+```
+You can define a title, bsStyle etc...
+
+A onSelect callback can be used 
+```javascript
+	function onItemSelect(object, event) {
+		if (onSelect) {
+			onSelect(event, object);
+		}
+	}
+```
+
+#### OVERLAY
+You can add an overlay to the main dropdown button. It will used TooltipTrigger from react-bootstrap.
+```javascript
+	if (hideLabel || tooltipLabel) {
+		return (
+			<TooltipTrigger label={tooltipLabel || label} tooltipPlacement={tooltipPlacement}>
+				{dropdown}
+			</TooltipTrigger>
+		);
+	}
+```
+
+
+#### DOPDOWN ITEMS
+Will choose the right function to render dropdown items (items or components props).
+If the props are empty, the func will return a MenuItem with no options label. 
+```javascript
+    function chooseMenuItemRendering(getComponent, items, components) {
+	if (
+		getComponent &&
+		Array.isArray(get(components, 'itemsDropdown')) &&
+		components.itemsDropdown.length > 0
+	) {
+		return components.itemsDropdown.map((component, index) =>
+			injectMenuItem(getComponent, component, index),
+		);
+	}
+	if (items.length > 0) {
+		return items.map(getMenuItem);
+	}
+	return <MenuItem disabled>No options</MenuItem>;
+}
+```
+## DROPDOWN ITEMS
+
+You have two different ways to defines this items. 
+Also there is some kind of special item, the divider. It will be a simple ```javascript { divider: true } ```, it will create a divider between your items.
+##### 1 ) With items props
+```javascript
```

Comment:
example is nice but tipical shape of item used by the component would be nice
example id are required when item is not a divider
label is used to display information to the user
etc...